### PR TITLE
Use screens instead of explicit `back` list

### PR DESCRIPTION
**Pull request by mgeplf**
* Instead of keeping track of state
* have a header to show which resource we're looking at

### CODE REVIEW

**Comment by jdcourcol** [comment]
I tried:
 `bluepyentity --bucket bbp/somatosensorycortex explorer "https://bbp.epfl.ch/neurosciencegraph/data/e0ad3eb2-0c4a-4097-9d99-5cc9625ab13f"`
and then go to that link,
image
 
then back and then try to go forward on the same link and got this:


image

**Comment by jdcourcol** [comment]
The issue was probably there earlier, but that is the first thing I tried :)

**Comment by mgeplf** [comment]
> The issue was probably there earlier, but that is the first thing I tried :)

No, I think it's a regression, I'll have a look sometime.

File: bluepyentity/app/explorer.py [hunk 1]
```diff
@@ -176,6 +178,8 @@ def hide_hints(self):
             for elem in elems:
                 # +2 because of the []
                 elem.right_crop(self._size_combination + 2)
+        self._cur_kt = self._initial_tree
```
Comment (mgeplf) [inline bluepyentity/app/explorer.py:181]: `self._initial_tree` is just an empty `_tree()`, right?

Comment (jdcourcol) [reply bluepyentity/app/explorer.py:181]: initially it is empty, then it is filled in with the keys structure.

Comment (mgeplf) [reply bluepyentity/app/explorer.py:181]: `self._initial_tree` is only assigned to in the `_init_state()`, right after `self._cur_kt = _tree()`, so maybe it's missing an update somewhere?

Comment (jdcourcol) [reply bluepyentity/app/explorer.py:181]: it is updated by "add_to_keytree". It is a reference to the root of the tree.

**Review by jdcourcol — APPROVED** [review]

**Pull request merged**